DateTime.js

```diff
@@ -5,11 +5,12 @@ var assign = require('object-assign'),
         createClass = require('create-react-class'),
 	moment = require('moment'),
 	React = require('react'),
+        onClickOutside = require('react-onclickoutside'),
```

Comment:
I am not sure I understand the implications of this change, and why it is necessary. Could you try to explain it further? Also this sounds like a major version update because (if I understand it correctly) it would break all current components, correct?

Reply:
Previously, the click handler was registered on the faux date input display, which caused the `onClickOutside` event to fire when clicking on the input itself. This change wraps the whole bundle in one click listener.

It potentially changes the behavior of the inputs, as it now wraps them all in the same click listener. I believe that the previous implementation was buggy, as the `onClickOutside` was actually being triggered on the first click _inside_.